### PR TITLE
Feature/space profile banner

### PR DESCRIPTION
**Pull request by nttuyen**
PR for space and profile banner feature
- This PR fix the bug that override group portal config

### CODE REVIEW

File: component/portal/src/main/java/org/exoplatform/portal/config/GroupPortalConfigListener.java [hunk 1]
```diff
@@ -110,7 +118,11 @@ public void preSave(Group group, boolean isNew) throws Exception {
             }
 
             // Create the portal from the template
-            portalConfigService.createGroupSite(groupId);
+            if (dataStorage.getPortalConfig(PortalConfig.GROUP_TYPE, groupId) == null) {
+                portalConfigService.createGroupSite(groupId);
```
Comment (thomasdelhomenie) [inline component/portal/src/main/java/org/exoplatform/portal/config/GroupPortalConfigListener.java]: Shouldn't sites be created in regards to the import mode ? I mean, if the import mode is set to CONSERVE for example, new site should not be created on an already populated instance.
What is the purpose of this change ?

Comment (nttuyen) [reply component/portal/src/main/java/org/exoplatform/portal/config/GroupPortalConfigListener.java]: - Shouldn't sites be created in regards to the import mode ? --> yes, this was done in UserPortalConfigService.start method, and its plugin (NewPortalConfigListener) --> this service and plugin use to create pre-configured portal config (start-up time)

- This GroupPortalConfigListener is use to create portal config in run-time, when we create new group, and this is a bug when we don't check for null, this listener will override the pre-configured portal config that was created by the UserPortalConfigService
